skrifa/src/patchmap.rs

```diff
@@ -293,6 +434,44 @@ impl PatchUri {
     }
 }
 
+/// Stores a materialized version of an IFT patchmap entry.
+///
+/// See: <https://w3c.github.io/IFT/Overview.html#patch-map-dfn>
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+struct Entry {
+    // Key
+    codepoints: IntSet<u32>,
+    feature_tags: BTreeSet<Tag>,
+    design_space: Vec<DesignSpaceSegment>,
+
+    // Value
+    uri: PatchUri,
+    compatibility_id: [u32; 4], // TODO: Make this a reference?
+}
+
+impl Entry {
+    fn new(template: &str, compat_id: &[u32; 4], default_encoding: &PatchEncoding) -> Entry {
+        Entry {
+            codepoints: IntSet::<u32>::empty(),
+            feature_tags: BTreeSet::<Tag>::new(),
```

Comment:
just fyi you don't need to specify types here, the compiler will figure it out. (If this is a matter of personal style, that's fine)